work-process/user-stories.md

```diff
@@ -2,26 +2,25 @@
 
 ## User Story overview
 
-- In Operations repository when you open an issue, it has a template that helps you define the User Story.
-- User Stories in this step must be moved into `Backlog` pipeline in ZenHub.
-- Various labels are added, see [Issue Labels](#issue-labels).
-- The Scrum Master and Product Owner verify if the User Story is well defined.
-  *  they assign people from whom they want to receive feedback for better definition of the User Story.
-  * Once these people post their feedback, they unassign themselves from the User Story.
-- Once the Scrum Master and Product Owner agree that the User Story is well defined, they add ``✋ [vote]`` prefix to the title of the User Story. At that point, Scrum Master and/or Product Owner assign people from whom they want to receive User Story Points estimation. With this, online poker planning begins.
-- At the end of the online poker planning, Story Points are added to the User Story.
-- The User Story is now prepared to be moved to the top of the User Story Pyramid stack.
-- On Sprint Planning meeting we choose which User Stories get included in the new sprint and we assign champions to them. From this point on, noone should be unassigned from the story so we know who worked on what when the sprint ends.
-- When the Champion deems the story done, they move it to the `Review` column. Scrum Master and Product Owner verify work was done as expected and move the Story to column `Done`.
+1. New issue is created using the User story template.
```

Comment:
User story -> User Story

Reply:
Yep it should be user story and the old habit is hard to break :stuck_out_tongue:  :+1: